src/test/resources/features/refapp-2.x/05-location/locations.feature

```diff
@@ -1 +1,29 @@
-Feature: Location Management
\ No newline at end of file
+Feature: Location Management
+
+  Background:
+    When a user clicks on configure metadata link from the home page
+    Then the system loads configure metadata dashboard
+
+  @selenium
+  @location
+  Scenario: LOCATION_MANAGEMENT
+    # User story: Add Location Attribute type
+    When a user clicks on the configure metadata link from the home page
+    Then the system loads configure metadata link dashboard
+    And a user clicks on Manage Location Attribute Types
+    And a user clicks on add new location attribute type
+    And a user fills the form
+    And the user saves the form
+    Then the form is saved
+
+    # User story: Add new location tag
+    When a user clicks on manage location tag
+    Then the user clicks on add new location
+    And a user fills add new location tag form
+    And a user saves add New location tag form
+    Then the New location tag form is saved
+
+    # User story: Manager locations
+    When a user clicks on manager locations
+    Then the user click on add new location
+    And a user fills add new location form
```

Comment:
did you forget to add an empty line to cater for git collaborators?

Reply:
@Ndacyayisenga-droid did you address the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/242/files#r756006352)?

Reply:
Feel free to address all the comments being reviewed. Thanks